draft-js-mention-plugin/src/mentionStrategy.js

```diff
@@ -1,12 +1,12 @@
 import { Entity } from 'draft-js';
 
-const findMention = (character) => {
+const findMention = (trigger) => (character) => {
   const entityKey = character.getEntity();
-  return (entityKey !== null && Entity.get(entityKey).getType() === 'mention');
+  return (entityKey !== null && Entity.get(entityKey).getType() === `mention-${trigger}`);
```

Comment:
I can't think of any use case in which a custom `type` is needed on its own.

Even if the `mentionTrigger` and `type` functionalities are separated, I believe it'd still be a good idea to make sure that the entity type doesn't conflict with other plugins or generic entity types, by prefixing the `type` string with some sort of mention plugin identifier string or by some other way?